centos-7.template

```diff
@@ -180,6 +180,10 @@ rm -f yum-wrapper.base64
 mv yum-wrapper /usr/bin/yum
 chmod +x /usr/bin/yum
 
+# Allow to run a custom script
+echo "test -f /opt/minishift-startup && /opt/minishift-startup" >> /etc/rc.d/rc.local
+
```

Comment:
Extra line I guess.

Reply:
Just a question @gbraad , How we get this file `/opt/minishift-startup` ?

Reply:
you create it...